docs/3.0/vsTestAdapter.html

```diff
@@ -0,0 +1,112 @@
+
+<p>The NUnit 3.0 Test Adapter allows you to run NUnit 3.0 tests inside Visual Studio. This is a new adapter, based partly on the code of the original NUnit Test Adapter, but modified to work with NUnit 3.0.</p>
+
+<p>It is not possible to run NUnit 2.x tests using this adapter. Use the original adapter for that purpose. If you need to work with projects using NUnit 2.x and other projects using NUnit 3.0, you may install both versions of the adapter.</p>
+
+<p>The current 3.0 Beta release is designed to work with Studio 2012 (All updates), and with Visual Studio 2013 (All updates including Update 3 checked September 2014). Some features are not available under VS2012 RTM.</p>
+
+<p>Releases of Visual Studio prior to VS 2012 did not have the ability to directly run tests built with Open Source testing frameworks like NUnit.</p>
+
+<h2>Installation</h2>
+
+<p>There are two ways of installing the adapter within Visual Studio. See Note 1 below for info on how to choose.</p>
+
+<ol>
+<li>Use the Extension Manager</li>
+<li>Add it as a Nuget package to your solution. (Requires VS 2012 Update 1 or later)</li>
+</ol>
+
+<h3>Installing With the Extension Manager</h3>
+
+<p>To install the NUnit Test Adapter using the Extension Manager, follow these steps:</p>
+
+<ol>
+<li>From within Visual Studio 2012 or 2013, select Tools | Extension Manager.</li>
+<li>In the left panel of the Extension Manager, select Online Extensions.</li>
+<li>Locate (search for) the NUnit 3.0 Test Adapter in the center panel and highlight it.</li>
+<li>Click 'Download' and follow the instructions.</li>
+</ol>
+
+<p>Use the Extension Manager to ensure that the NUnit 3.0 Test Adapter is enabled.</p>
+
+<h3>Installing the Nuget Package</h3>
+
+<p>To add it is a Nuget package, you must have an active solution, then follow these steps:</p>
+
+<ol>
+<li>From Tools menu, use Library Package Manager and select Manage NuGet packages for solution</li>
+<li>In the left panel, select Online</li>
+<li>Locate (search for) NUnit 3.0 Test Adapter in the center panel and highlight it</li>
+<li>You will find two packages, one without framework and one with framework. See note 2 below for which to choose.</li>
+<li>Click 'Install'</li>
+<li>In the "Select Projects" dialog, you need to select at least one project to add the adapter to, see notes below.</li>
+</ol>
+
+<h3>Notes</h3>
+
+<p><strong>Note 1: How to choose between Extension and NuGet package</strong></p>
+
+<p>The Extension will apply to Visual Studio itself, and will work for all projects you use. All users of your solution need to install the Extension. If you use TFS Build you must also install the extension to the build system there.
+The Package will apply to the solution, and will work for any other user too, as it follows the solution, but requires the user to have Update 1 or above. It will also work for any TFS2012 Update 1 or above server build, including TF Service and requires no further installation.</p>

```

Comment:
I'll fix the first. I think it belongs here as counterpoint to the previous paragraph.